README.md

```diff
@@ -1 +1,167 @@
-# land-rover-back-end
\ No newline at end of file
+# Land Rover E-Commerce ![Rails](https://rubyonrails.org/assets/images/logo.svg)
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Front-End React](#frontend)
+    - [Key Features](#key-features)
+- [💻 Getting Started](#getting-started)
+  - [Prerequisites](#prerequisites)
+  - [Setup](#setup)
+  - [Install](#install)
+  - [Database](#database)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [🙏 Credits](#credit)
+- [📝 License](#license)
+
+# 📖 Land Rover E-Commerce <a name="about-project"></a>
+
+**Land Rover E-Commerce** The Final Capstone Project is based on an E-commerce app to make a reservation to try Land Rover vehicle.
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+### Front End React <a name="#frontend"></a>
+
+- [Front-End React](https://github.com/VelzckC0D3/Land-Rover-E-Commerce) 
+
+### Key Features <a name="key-features"></a>
+
+- Devise Authentication
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running follow these simple example steps.
+
+### Prerequisites
+
+In order to run this project you need:
+
+- [Ruby](https://www.ruby-lang.org/en/) installed and running
+- [Rails](https://rubyonrails.org/) installed and running
+- [PostgreSQL](https://www.postgresql.org/) installed and running
+
+### Setup
+
+Clone this repository to your desired folder:
+
+```
+git clone https://github.com/VelzckC0D3/land-rover-back-end.git
+cd land-rover-back-end
+code .
+```
+
+### Install
+
+Install the project dependencies, execute the following command in Bash:
+
+```sh  
+  bundle install
+```
+
+### Database
+
+Create the database, execute the following command in Bash:
+
+```sh
+rails db:create
```

Comment:
- [ ] If I set the project according to the instructions, I can't create the database:

![image](https://github.com/VelzckC0D3/land-rover-back-end/assets/57421823/b6439529-af91-40d7-87ae-17639292f11f)

This is because your users need to create the master key locally.

Kindly, add the following instructions to your "install" section:

```text
1. Remove config/master.key and config/credentials.yml.enc if they exist.
2. Run in the terminal: EDITOR=code rails credentials:edit 
3. Close the editor that opens.
4. This command will create a new master.key and credentials.yml.enc if they do not exist.
```

Result:

![image](https://github.com/VelzckC0D3/land-rover-back-end/assets/57421823/66feeca5-6dd9-4952-9665-33dea07d3cbf)

That way, your new users will be able to generate their own secrets, and be able to create the database. 💪